### PR TITLE
Add reason to unsilence command

### PR DESCRIPTION
**Pull request by minisbett**
One of the things I added on cs0su for convenience, would also make it consistent as there are also reasons for unrestrictions

### CODE REVIEW

**Comment by cmyui** [comment]
@osuAkatsuki/bancho-py-devs think there's value to this?

**Comment by minisbett** [comment]
> @osuAkatsuki/bancho-py-devs think there's value to this?

Idk maybe there's a usage, pretty much the same there'd be for restrictions. Would be weird if it's on !unrestrict but not !unsilence

File: app/commands.py [hunk 1]
```diff
@@ -826,8 +826,8 @@ async def unsilence(ctx: Context) -> Optional[str]:
     if target.priv & Privileges.STAFF and not ctx.player.priv & Privileges.DEVELOPER:
         return "Only developers can manage staff members."
 
-    await target.unsilence(ctx.player)
-    return f"{target} was unsilenced."
+    await t.unsilence(ctx.player, " ".join(ctx.args[1:]))
```
Comment (NiceAesth) [inline app/commands.py:829]: huh?

Comment (minisbett) [reply app/commands.py:829]: oh I fucked up the merge conflict 1 sec

Comment (cmyui) [reply app/commands.py:829]: we should at a minimum
```suggestion
    await t.unsilence(ctx.player, reason=" ".join(ctx.args[1:]))
```

Comment (minisbett) [reply app/commands.py:829]: why that

Comment (minisbett) [reply app/commands.py:829]: unrestrict doesn't have it either

**Comment by cmyui** [comment]
> Idk maybe there's a usage, pretty much the same there'd be for unrestrictions. Would be weird if it's on !unrestrict but not !unsilence

i think there was already limited value to unrestrict message (generally was used to say "appealed" on akatsuki, but was already getting spammed with like "a" and "xd" and stuff), but for unsilence.. really seems like we're picking at straws

File: app/commands.py (hunk 1 above)
Comment (NiceAesth) [reply app/commands.py:829]: speaking of unrestrict, i am not a fan of having the join inline. would rather you put in in a variable

Comment (minisbett) [reply app/commands.py]: fixed

**Comment by NiceAesth** [comment]
well, i am fine with the impl, but i am kinda indifferent towards whether or not this is needed. would need other opinions

**Review by cmyui — APPROVED** [review]

**Pull request merged**